cookbooks/magnetic_stripes/magnetic_stripes.cc

```diff
@@ -113,7 +116,17 @@ namespace aspect
   {
     ASPECT_REGISTER_MATERIAL_MODEL(MagneticStripes,
                                    "magnetic stripes",
-                                   "A simple material model that is like the "
-                                   "'melt simple' model, but has a constant reaction term.")
+                                   "A simple material model that is like the 'composition "
+                                   "reaction' model, but with a different reaction term. "
+                                   "In areas above a depth of 7 km and with a velocity that "
+                                   "does not diverge from vertically upwards by more than 45 "
+                                   "degrees, the first compositional field used in the model "
+                                   "is set to either one or minus one. The sign depends on a "
+                                   "list of reversal times that is passed to the material "
+                                   "model as an input. Bevor the 1st, 3rd, 5th, ... entry, "
```

Comment:
`Bevor` -> `Before`

Reply:
Thanks for catching this! That would have been the German way to write this, I was definitely too tired...